src/results/financial.jl

```diff
@@ -356,7 +356,7 @@ function get_depreciation_schedule(p::REoptInputs, tech::Union{AbstractTech,Abst
 
     federal_itc_fraction = 0.0
     try 
-        federal_itc_fraction = tech.federal_itc_fraction
+        federal_itc_fraction = tech.federal_itc_fraction # TODO: also check for total_itc_fraction as storage does not use federal_itc_fraction?
```

Comment:
@Bill-Becker  this doesn't need to be part of this PR, but it looks like you updated this function to include storage. However, the storage itc input has a different name, so the warning below will always appear for storage and it looks like itc will be set to 0 in determining the storage depreciation schedule?

Reply:
Bill is fixing in separate PR